bio/strling/call/meta.yaml

```
@@ -0,0 +1,4 @@
+name: "strling call"
+description: STRling (pronounced like “sterling”) is a method to detect large STR expansions from short-read sequencing data. "Call" calls genotypes/estimate allele sizes for all loci in each sample. (https://strling.readthedocs.io/en/latest/run.html)
```

Ah, forgot the suggestion here:
```suggestion
description: >
    STRling (pronounced like “sterling”) is a method to detect large short tandem repeat (STR) expansions from short-read sequencing data.
    ``call`` calls genotypes/estimate allele sizes for all loci in each sample.
    Documentation at: https://strling.readthedocs.io/en/latest/run.html
```